crates/macros/src/build_macro.rs

```diff
@@ -70,10 +70,7 @@ impl Parse for BuildMacro {
                         if let Some((namespace, name)) = reader.get_type_name(namespace, &name) {
                             build.import(namespace, name);
                         } else {
-                            return Err(Error::new_spanned(
-                                input,
-                                format!("`{}.{}` not found in metadata", namespace, name),
-                            ));
+                            return Err(Error::new_spanned(input, "Type not found in metadata"));
```

Comment:
Despite spanning, I think the name should still be printed as part of the error.

Take for example a similar error in Rust code:

```log
error[E0433]: failed to resolve: use of undeclared crate or module `foo`
```

That way an annotations processor - like GH CI - picks out `^error` lines and prints them in full; with this change the name context is now lost.